lms/djangoapps/verify_student/tasks.py

```diff
@@ -114,11 +114,12 @@ def send_request_to_ss_for_user(self, user_verification_id, copy_id_photo_from):
     log.info('=>New Verification Task Received %r', user_verification.user.username)
     try:
         headers, body = user_verification.create_request(copy_id_photo_from)
+        # checkout PROD-1395 for detail why we are adding system certificate paths for verification.
```

Comment:
I looked at the ticket this comment references, but it doesn't clarify why we're specifying a single file path for verification rather than relying on the default behavior of `requests` to do SSL verification using the trusted CA bundle.

Reply:
@matthugs as I mentioned in the description SSL certificate chain is broken. So we are downloading the intermediate certificate and adding it to system CA bundle. System and requests CA bundles are same both have trusted root certificates. 
We have updated the system CA bundle that's why we are asking requests to use our system CA bundle instead of requests default.